tests/cyclops/query/test_omop.py

```diff
@@ -6,8 +6,8 @@
 
 
 @pytest.mark.integration_test
-def test_omop_querier():
-    """Test OMOPQuerier."""
+def test_omop_querier_synthea():
+    """Test OMOPQuerier on synthea data."""
     synthea = OMOPQuerier("cdm_synthea10", database="synthea_integration_test")
```

Comment:
Some code duplication going on here. Extract the OMOPQuerier into its own fixture?